koku/masu/database/trino_sql/reporting_ocpusagelineitem_daily_summary.sql

```diff
@@ -397,6 +401,7 @@ FROM (
             )
         ) as volume_labels,
         sli.source as source_uuid,
+        max(sli.csi_volume_handle) as csi_volume_handle,
```

Comment:
Let's move this into the `group by` instead of using a `max`. It would be more consistent with the storageclass, pvc, pv, node, etc. in this query.

Reply:
In my opinion the current [group by options](https://github.com/project-koku/koku/blob/0acfe707aefed67e46fb9ec6baebddd9f83c19c4/koku/masu/database/trino_sql/reporting_ocpusagelineitem_daily_summary.sql#L434-L443) for that query provide enough granularity that a max is sufficient here.